authentication.go

```diff
@@ -46,6 +46,7 @@ type Token struct {
 type AuthURLParams struct {
 	ForceApproval bool
 	State         string
+	Flags         string
```

Comment:
in the other SDKs I made `Flags` a `[]string`, so we might want to change to this as well here!